tests/Microsoft.DotNet.Framework.Docker.Tests/projects/webapp-4.6.2/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG BASE_BUILD_IMAGE
+
+FROM $BASE_BUILD_IMAGE
+
+WORKDIR /app
+COPY . ./
+
+RUN nuget restore SimpleWebApplication.csproj -SolutionDirectory -NoCache
+
+RUN msbuild SimpleWebApplication.csproj /p:Configuration=Release
+
+RUN dir ./bin/SimpleWebApplication.dll
```

Comment:
Per our discussion, I think the verification (e.g. `dir ./bin/SimpleWebApplication.dll`) logic best belongs in the test case.

Reply:
I gave this more thoughts.  CMD is probably the most appropriate, which is executed during "docker run", that will be the test case you are referring.

Reply:
I don't understand why it would be most appropriate for the CMD to be `dir ./bin/SimpleWebApplication.dll`.  Please explain.  This doesn't seem like a normal usage.  I would prefer the Dockerfile to be a typical usage we expect to see.  All verification logic should reside within the tests.  Please fix so that we can merge this PR.

Reply:
using CMD, the existence of the dll after build is being verified at docker run time.  The verification logic is in the testcase, reside within the tests.  Hopefully this answers your questions!

Reply:
Why is using the CMD most appropriate over using `docker run` on the resulting image.  Per my original comments this is not going to be typical usage.  Please change to place the verification logic in the test case.